content/en/blog/releases/release-0.7.0.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Release 0.7.0"
+linkTitle: "Release 0.7.0"
+date: 2022-07-20
+description: >
+  What is new in version 0.7.0?
+---
+
+**We are pleased to announce the release of Merbridge 0.7.0!**
+
+This release mainly includes two updates as below:
+
+1. Replaced XDP with `tc` (Traffic Control) to handle the ingress/egress traffic. This replacement comes from a situation that the XDP Generic mode has some problems and is not recommended to use in a production environment.
+
+2. With the support of [@bartsmykla](https://github.com/bartsmykla), an engineer from [Kuma](https://kuma.io/), Merbridge now has well supported for Kuma, a universal Envoy service mesh. Similar to Istio, you can use all capabilities provided by Merbridge when you are working in the Kuma environment.
```

Comment:
Maybe we can change the order to emphasize this new feature: 
Merbridge now supports Kuma, a universal Envoy service mesh. Similar to Istio, you can use all capabilities provided by Merbridge when you are working in the Kuma environment. Thanks [@bartsmykla](https://github.com/bartsmykla), an engineer from [Kuma](https://kuma.io/).